Week3/HW3/3_moreJavascript/4_vehicleType.js

```diff
@@ -0,0 +1,13 @@
+function vehicleType(color,code) {
+    if (code ===1) {
+        type = "car"
+    } else if 
+    (code === 2) {
```

Comment:
Slight indentation problem here with else if and the parantheses and also you missed using a keyword while creating variable `type` 😱

Reply:
i think it is better now.